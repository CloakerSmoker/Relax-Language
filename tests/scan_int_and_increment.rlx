#Require "$/String.rlx"
#Require "$/Console.rlx"

/*
ARGS: 123 -20 10000000000
*/

define i32 Main(i64 ArgC, i8** ArgV) {
	GetArgs(&ArgC, &ArgV)
	
	for (i32 Index := 1, Index < ArgC, Index++) {
		i64 N := AToI(ArgV[Index])
		
		Print(N + 1)
		Print(" ")
	}
	
	return 0
}
