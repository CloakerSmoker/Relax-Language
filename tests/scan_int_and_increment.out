124 -19 10000000001 